rotkehlchen/db/dbhandler.py

```diff
@@ -2379,6 +2392,35 @@ def get_binance_pairs(self, name: str, location: Location) -> List[str]:
             return json.loads(data[0])
         return []
 
+    def set_ftx_subaccount(self, name: str, subaccount: str) -> None:
```

Comment:
```suggestion
    def set_ftx_subaccount(self, ftx_name: str, subaccount_name: str) -> None:
```

Better var names. I was wondering what the heck `name` was.